### DISCUSSION TITLE
Build.md is missing

### DISCUSSION THREAD

**Discussion by Magos-crawler** [dlemstra/Magick.NET#1736] [Help] [answered]
Is this how it was intended or am I still missing something?

![opera_x2ao01sslr](https://github.com/user-attachments/assets/94e2b11f-d569-4510-888e-5bba70133947)
![opera_C5Cbmz375w](https://github.com/user-attachments/assets/b6a6f732-8385-4368-a0c8-e5af45e0ab6a)

**Comment by dlemstra** [comment, marked as answer]
Thanks for pointing this out. I have fixed this.
